src/views/Nft/market/Profile/components/EditProfileModal/ChangeProfilePicView.tsx

```diff
@@ -78,30 +78,33 @@ const ChangeProfilePicPage: React.FC<ChangeProfilePicPageProps> = ({ onDismiss,
       <Text as="p" color="textSubtle" mb="24px">
         {t('Choose a new Collectible to use as your profile pic.')}
       </Text>
-      <Box maxHeight="300px" overflowY="scroll">
-        {nftsInWallet.map((walletNft) => {
-          const handleChange = () => {
-            setSelectedNft({
-              tokenId: walletNft.tokenId,
-              collectionAddress: walletNft.collectionAddress,
-            })
-          }
-          return (
-            <SelectionCard
-              name="profilePicture"
-              key={`${walletNft.collectionAddress}#${walletNft.tokenId}`}
-              value={walletNft.tokenId}
-              image={walletNft.image.thumbnail}
-              isChecked={walletNft.tokenId === selectedNft.tokenId}
-              onChange={handleChange}
-              disabled={isApproving || isConfirming || isConfirmed}
-            >
-              <Text bold>{walletNft.name}</Text>
-            </SelectionCard>
-          )
-        })}
-      </Box>
-      {nfts.length === 0 && (
```

Comment:
Root cause of the issue is here, it uses nfts instead of nftsInWallet to determine showing warnings